aldryn_people/models.py

```diff
@@ -1,43 +1,55 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import unicode_literals
+
 import base64
+import six
 import urlparse
-import warnings
 import vobject
+import warnings
 
-import six
-
-from django.core.urlresolvers import reverse
 from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.core.urlresolvers import reverse
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
-from django.utils.translation import ugettext_lazy as _
+try:
+    from django.utils.encoding import force_unicode
+except ImportError:
+    from django.utils.encoding import force_text as force_unicode
 try:
     from django.utils.text import slugify
 except ImportError:
-    # In Djano <= 1.4 slugify is here
+    # In Django <= 1.4 slugify is here
     from django.template.defaultfilters import slugify
+from django.utils.text import slugify as default_slugify
+from django.utils.translation import ugettext_lazy as _, ugettext, override
 
+from aldryn_common.slugs import unique_slugify
+from aldryn_common.admin_fields.sortedm2m import SortedM2MModelField
 from cms.models.pluginmodel import CMSPlugin
-
+from cms.utils.i18n import get_current_language
 from djangocms_text_ckeditor.fields import HTMLField
-
 from filer.fields.image import FilerImageField
-
 from parler.models import TranslatableModel, TranslatedFields
 
-from aldryn_common.slugs import unique_slugify
-from aldryn_common.admin_fields.sortedm2m import SortedM2MModelField
-
 from .utils import get_additional_styles
 
+if settings.LANGUAGES:
```

Comment:
two points:

* I don't agree with this code being repeated on multiple addons, django comes with this setting already so don't think is necessary to be checking.
* ```settings.Language``` does not exist and will raise ```AttributeError```, guess you're looking for ```LANGUAGE_CODE```